tests/cli/commands/test_connection_command.py

```diff
@@ -34,13 +32,11 @@
 from tests.test_utils.db import clear_db_connections
 
 
-class TestCliGetConnection(unittest.TestCase):
-    def setUp(self):
-        self.parser = cli_parser.get_parser()
-        clear_db_connections()
+class TestCliGetConnection:
+    parser = cli_parser.get_parser()
 
-    def tearDown(self):
-        clear_db_connections()
+    def setup_method(self):
+        clear_db_connections(add_default_connections_back=True)
```

Comment:
I think we should do this after the test to clean up properly. Maybe a `setup_class` and a `teardown_method` so we ensure a clean slate both before and after each test.

Reply:
yeah i was thinking about that

maybe an autouse fixture in the module that clears after yield

they could all use same fixture except that only some of the classes require the example connectios

Reply:
like
```

Reply:
like
```
@pytest.fixture(scope='class', autouse=True)
def clear_connections():
    yield
    clear_db_connections(add_default_connections_back=False)

```

Reply:
done

Reply:
Isn’t that essentially the same as

```python
def teardown_method(self):
    clear_db_connections(add_default_connections_back=False)
```

but more code?

Reply:
no because there are 4 (or more?) classes and this will apply to all of them... right?

Reply:
actually 6 :)